README.md

```diff
@@ -92,6 +92,28 @@ This has a single build target `snmalloc_lib`, which includes
 the necessary compiler and linker flags, to use snmalloc as a header-only
 library.
 
+# Using snmalloc as header-only library
+
+In this section we show how to compile snmalloc into your project such that it replaces the standard allocator functions such as free and malloc. The following instructions were tested with CMake and Clang running on Ubuntu 18.04.
+
+Add these files to your CMake file
+```
+set(SNMALLOC_ONLY_HEADER_LIBRARY ON)
+add_subdirectory(snmalloc EXCLUDE_FROM_ALL)
+```
+
+In addition make sure you executable is compiled to support 128 bit atomic operations. This may require you to add the following to your CMake file.
+```
+target_compile_options([lib_name] PRIVATE -mcx16)
```

Comment:
So I think this should really be
```
target_link_libraries([my_applib] PUBLIC execinfo)
```
Or for a header only library you are adding snmalloc to 
```
target_link_libraries([my_applib] INTERFACE execinfo)
```
Did that not include the `-mcx16` setting on you subsequent target?

Reply:
I updated the PR